libraries/lib-screen-geometry/ViewInfo.cpp

```diff
@@ -325,25 +326,24 @@ ProjectFileIORegistry::AttributeReaderEntries entries {
 {
    return ViewInfo::Get(project).selectedRegion;
 },
-NotifyingSelectedRegion::Mutators(L"sel0", L"sel1")
+NotifyingSelectedRegion::Mutators("sel0", "sel1")
 };
 
 ProjectFileIORegistry::AttributeReaderEntries entries2 {
 // Just a pointer to function, but needing overload resolution as non-const:
 (ViewInfo& (*)(AudacityProject &)) &ViewInfo::Get,
 {
-   { L"vpos", [](auto &viewInfo, auto value){
-      long longVpos;
-      wxString(value).ToLong(&longVpos);
+   { "vpos", [](auto &viewInfo, auto value){
+      long longVpos = value.GetInt32();
```

Comment:
Is this a not always correct assumption that sizeof(long) is sizeof(int32_t)?

Reply:
On our platforms  - yes, but let me rewrite it in a safer way.

Reply:
Hmm, I've looked below, we convert it to `int` later on

Reply:
Actually, previous code could lead to an undefined value in viewInfo.vpos. 

`ToLong` docs state `false if the string does not represent a valid number in the given base (the value of val may still be modified in this case).`

Reply:
Just to clarify - I've tried to keep the diffs as small as possible, and I guess it was your intention as well when you were extracting this code into a library.

Reply:
What's worse, now I see that in some cases (see `zoom` nearby the original code had undefined behavior and I'm no longer sure that my changes are equivalent. I think I will go ahead with that TryGet refactoring, this will force me to rewrite all such code as `a = value.Get(a)`.